doctrine/custom_dql_functions.rst

```diff
@@ -146,4 +146,4 @@ In Symfony, you can register your custom DQL functions as follows:
                 ],
             ]);
 
-.. _`DQL User Defined Functions`: https://www.doctrine-project.org/projects/doctrine-orm/en/current/cookbook/dql-user-defined-functions.html
+.. _DQL User Defined Functions: https://www.doctrine-project.org/projects/doctrine-orm/en/current/cookbook/dql-user-defined-functions.html
```

Comment:
This change looks wrong to me.

Reply:
Looks strange to me too, but the current syntax isn't working either ;-) Is the parser broken?

Reply:
Maybe it is not working because of the other line in your PR

Reply:
No! The real question is: Why is it not working on https://symfony.com/doc/4.4/doctrine/custom_dql_functions.html **right now?**

Reply:
I think the error comes from the other content that was changed in this PR.

About the link syntax to define the URL, both work but in Symfony Docs we prefer the one that wraps the entire link text with backticks. So, let's revert this second change. Thanks!